include/fmt/os.h

```diff
@@ -391,7 +391,7 @@ struct ostream_params {
 };
 }  // namespace detail
 
-static constexpr detail::buffer_size buffer_size;
+static constexpr detail::buffer_size buffer_size{};
```

Comment:
Please add a short comment clarifying that {} is a workaround for gcc 8, preferably with a link to the bug report.